.github/workflows/code-quality.yml

```diff
@@ -0,0 +1,37 @@
+name: Code quality
+on:
+  push:
```

Comment:
add also
```
  create:
```
in addition to `push:` and `pull_request:`, so actions are run also when we branch-out a new branch in the main repo